harness/tests/test/unit/test_array.gd

```diff
@@ -51,6 +51,12 @@ func test_array_typed_not_null_remove():
 	assert_eq(invocation_script.nav_meshes_size(), 0, "Array size should be 0")
 	invocation_script.free()
 
+func test_create_user_type_variant_array():
+	var invocation_script = godot_tests_Invocation.new()
+	var user_type_variant_array = invocation_script.create_variant_array_of_user_type()
+	assert_not_null(user_type_variant_array, "Should return an empty user typed variant array.")
```

Comment:
Shouldn't we also check the actual type here? So we're sure that whatever object array we receive here has the actual types we expect?

Reply:
You can't check type of another language in gdscript

Reply:
Hmm but IIRC you can do this: `if obj extends Class`
So theoretically we should be able to do `assert_true(user_type_variant_array[0] extends godot_tests_subpackage_OtherScript, "Bla bla")` no?

Reply:
Nop, we cannot as right operand is a `KotlinScript`

Reply:
Hmm i always though also scripts written in other languages land in the class db because of the registering.
Well if this is not the case, I'm fine with it